src/Ledger/Utxo.lagda

```diff
@@ -7,33 +7,25 @@
 {-# OPTIONS --safe #-}
 {-# OPTIONS --overlapping-instances #-}
 
-open import Ledger.Transaction
+open import Ledger.Transaction using (TransactionStructure)
 
 module Ledger.Utxo (txs : TransactionStructure) where
 
 open import Ledger.Prelude hiding (Dec₁)
 
-open import Algebra using (CommutativeMonoid)
-open import Algebra.Structures
-open import Data.Integer using (ℤ; _⊖_)
```

Comment:
We also don't use `Integer`, though we do use `Integer.Ext` (but just `posPart` and `negPart` from that module).